mobile/android/app/build.gradle

```diff
@@ -34,7 +34,7 @@ android {
     defaultConfig {
         applicationId "com.example.devicemanager"
         minSdkVersion 21
-        targetSdkVersion 27
+        targetSdkVersion 29
         versionCode flutterVersionCode.toInteger()
         versionName flutterVersionName
         testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
```

Comment:
This should be updated to `androidx.test.runner.AndroidJUnitRunner`

Reply:
ok, thanks